tests/test_event.py

```diff
@@ -44,11 +45,18 @@ def test_decorator_registration():
     event_emitter = EventEmitter()
     registrar = EventRegistrar(event_emitter)
 
-    called = [False]
+    called = [False, False]
 
+    # Multiple handlers can be defined for the same event.
     @registrar.before_send_request
     def foo():
         called[0] = True
 
+    @registrar.before_send_request
+    def bar():
+        called[1] = True
+
     event_emitter.fire(EventType.before_send_request)
+
     assert called[0] is True
+    assert called[1] is True
```

Comment:
You can use mocks to assert something was called instead of this.

    hook1 = mock.Mock()
    registrar.before_send_request(hook1)
    hook1.assert_called_once_with()